test/event_test.py

```diff
@@ -28,6 +28,70 @@
 )
 
 
+names_and_events = (
```

Comment:
This should probably be `NAMES_AND_EVENTS` because it's a constant

Reply:
Thanks for the comment. I've made the requested change.